### PR TITLE
Add Preview Time on selection of beatmap

### PR DESCRIPTION
**Pull request by ColdVolcano**

### CODE REVIEW

File: osu.Game/Screens/Select/PlaySongSelect.cs [hunk 1]
```diff
@@ -279,20 +279,28 @@ private void selectBeatmap(BeatmapInfo beatmap)
...
                 if (beatmap.BeatmapSetID == Beatmap?.BeatmapInfo.BeatmapSetID)
+                {
                     sampleChangeDifficulty.Play();
+                    beatmapSetChange = false;
+                }
                 else
+                {
                     sampleChangeBeatmap.Play();
+                    beatmapSetChange = true;
+
```
Comment (SirCmpwn) [inline osu.Game/Screens/Select/PlaySongSelect.cs]: Extra newline here

File: osu.Game/Screens/Select/PlaySongSelect.cs [hunk 2]
```diff
@@ -279,20 +279,28 @@ private void selectBeatmap(BeatmapInfo beatmap)
...
+                }
 
                 Beatmap = database.GetWorkingBeatmap(beatmap, Beatmap);
             }
-
-            ensurePlayingSelected();
+
```
Comment (SirCmpwn) [inline osu.Game/Screens/Select/PlaySongSelect.cs]: Added whitespace here

**Comment by SirCmpwn** [comment]
+1, nitpicks aside

File: osu.Game/Screens/Select/PlaySongSelect.cs [hunk 3]
```diff
@@ -279,20 +279,28 @@ private void selectBeatmap(BeatmapInfo beatmap)
         /// </summary>
         private void selectionChanged(BeatmapGroup group, BeatmapInfo beatmap)
         {
+            bool beatmapSetChange = false;
             if (!beatmap.Equals(Beatmap?.BeatmapInfo))
             {
                 if (beatmap.BeatmapSetID == Beatmap?.BeatmapInfo.BeatmapSetID)
+                {
                     sampleChangeDifficulty.Play();
+                    beatmapSetChange = false;
```
Comment (RemieRichards) [inline osu.Game/Screens/Select/PlaySongSelect.cs]: this line isn't necessary

**Comment by peppy** [comment]
needs async load and fade out / fade in, but i guess this is a step in the right direction.

**Pull request merged**